### ISSUE TITLE
command-instead-of-module "yum clean" revisited

### ISSUE THREAD

**Issue by bluikko** [ansible/ansible-lint#1737]
##### Summary

The lint error `command-instead-of-module` is raised for command `yum clean`. As the `yum` module documentation states there is no support for doing `yum clean` operation in the module. So the only option is to run a command.

This issue has been raised in the past as #369 and `warn: false` was recommended. But that is now deprecated in 2.11.

So shouldn't ansible-lint detect if `yum clean` is being run and not raise `command-instead-of-module`?

If there is positive feedback and nobody else wants to do it then I can volunteer to make the patch - it should be a simple change, no?

##### Issue Type

- Bug Report

##### Ansible and Ansible Lint details

```console (paste below)
ansible --version
ansible [core 2.11.5]
  config file = /etc/ansible/ansible.cfg
  configured module search path = ['/usr/share/ansible/plugins/modules']
  ansible python module location = /opt/ansible/lib64/python3.6/site-packages/ansible
  ansible collection location = /usr/share/ansible/collections
  executable location = /opt/ansible/bin/ansible
  python version = 3.6.8 (default, Nov 16 2020, 16:55:22) [GCC 4.8.5 20150623 (Red Hat 4.8.5-44)]
  jinja version = 3.0.2
  libyaml = True

ansible-lint --version
ansible-lint 5.2.0 using ansible 2.11.5

```

- ansible installation method: pip
- ansible-lint installation method: pip

##### OS / ENVIRONMENT

EL7

##### STEPS TO REPRODUCE

Have a command that runs `yum clean`.

```yaml
  command:
    cmd: yum clean all
```

##### Desired Behaviour

`command-instead-of-module` should not be raised for `yum clean` which cannot be done using the `yum` module.

Possible security bugs should be reported via email to `security@ansible.com`

##### Actual Behaviour

`command-instead-of-module` is raised even though the module cannot do this action.

```paste below

```

[minimum complete verifiable example]: http://stackoverflow.com/help/mcve
